content/applications/sales/crm/track_leads/prospect_visits.rst

```diff
@@ -1,55 +1,116 @@
-===========================
-Track your prospects visits
-===========================
+====================
+Track webpage visits
+====================
 
-Tracking your website pages will give you much more information about
-the interests of your website visitors.
+Tracking the number of visits to individual webpages provides valuable insight about the interests
+of website visitors. It can also be used to analyze the success of marketing campaigns.
 
-Every tracked page they visit will be recorded on your lead/opportunity
-if they use the contact form on your website.
+With Odoo, tracking links can be created to track visits for any page on the website. When a lead is
+created from a website contact form, every page that the prospective customer visits is recorded on
+the lead/opportunity form in the *CRM* app.
 
-Configuration
-=============
+.. _crm/track-leads/link-tracker:
 
-To use this feature, install the free module *Lead Scoring* under your
-*Apps* page (only available in Odoo Enterprise).
+Track a webpage
+===============
+
+Odoo's *link tracker* creates a unique tracking link for any page on the company website. The link
+contains analytics trackers that can be used in Google Analytics to track clicks and visitors, and
+in Odoo reports to track opportunities and revenue. Send this link as part of a marketing campaign
+to track clicks and webpage views for that particular campaign.
 
-.. image:: prospect_visits/prospect_visits01.png
+To set up tracking for a webpage, go to any static page on the website through the
+:guilabel:`Website` app. Then click on the :guilabel:`Site` header menu at the top of the page, and
+select :guilabel:`Link Tracker`.
+
+.. image:: prospect_visits/link-tracker-menu.png
    :align: center
+   :alt: Link tracker selection from the Site header menu.
 
-Track a webpage
-===============
+On the Link Tracker page, the :guilabel:`URL` field is already filled in based on the webpage that
+was open when :guilabel:`Link Tracker` was clicked. However, the URL can be changed to any page on
+the website.
 
-Go to any static page you want to track on your website and under the
-*Promote* tab you will find *Optimize SEO*
+In the :guilabel:`Campaign`, :guilabel:`Medium`, and :guilabel:`Source` fields, specify a marketing
+campaign to associate with the tracking link. This information appears in the :guilabel:`Marketing`
+section of the :guilabel:`Extra Information` tab of a lead or opportunity form.
```

Comment:
```suggestion
In the :guilabel:`Campaign` field, specifiy a marketing campaign to associate with the tracking
link.

In the :guilabel:`Medium` field, define the specific medium used to share the link, like an email,
for example.

In the :guilabel:`Source` field, define the source from which the traffic comes from, like from
Facebook, for instance.

This information appears in the :guilabel:`Marketing` section of the :guilabel:`Extra Information`
tab of a lead or opportunity form.
```
Correct me if I'm wrong, but I don't believe each of these fields corresponds to marketing campaigns. The first one, Campaign, for sure does -- but 'Medium' and 'Source' represent different things, I believe.

Reply:
I think you're right! The Medium and Source are related to the Campaign, but I think I was over-generalizing, and your explanation is more accurate.